riscv/src/compiler.rs

```diff
@@ -1358,6 +1394,11 @@ fn process_instruction(instr: &str, args: &[Argument]) -> Vec<String> {
             only_if_no_write_to_zero(format!("{rd} <== load_label({label});"), rd)
         }
 
+        // The powdr embedded `return` instruction
+        "return" => {
```

Comment:
I don't think RISCV has an instruction called `return`

Reply:
Right.. I added it so that when I replace
```
tail main;
```
by
```
call main;
return;
```

The `return` gets turned into powdr asm. I guess I should instead run this substitution on the powdr asm.

Reply:
yea I think so too

Reply:
moved to the asm to powdr asm match